File: src/c1removeadjacentduplicates/Go/main.go
package main

import (
	"fmt"
	"strings"
)

func main() {
	inputs := []string{
		"g",
		"ggaabcdeb",
		"abbddaccaaabcd",
		"aannkwwwkkkwna",
		"abbabccblkklu",
	}

	for i, input := range inputs {
		fmt.Printf("%d.\tRemove duplicates from string: \"%s\"\n", i+1, input)
		resultingString := removeDuplicates(input)
		fmt.Printf("\tString after removing duplicates: \"%s\"\n", resultingString)
		fmt.Printf("%s\n", strings.Repeat("-", 100))
	}
}

File: src/c1removeadjacentduplicates/JS/main.js
const removeDuplicates = require('./removeDuplicates');

const inputs = [
    "g",
    "ggaabcdeb",
    "abbddaccaaabcd",
    "aannkwwwkkkwna",
    "abbabccblkklu"
];

inputs.forEach((input, i) => {
    console.log(`${i + 1}.\tRemove duplicates from string: "${input}"`);
    const resultingString = removeDuplicates(input);
    console.log(`\tString after removing duplicates: "${resultingString}"`);
    console.log('-'.repeat(100));
});
